application/config/database.php

```diff
@@ -37,13 +37,13 @@
 $active_group = "default";
 $active_record = TRUE;
 
-$db['default']['hostname'] = "localhost";
+$db['default']['hostname'] = "127.0.0.1";
 
 // MySQL
-$db['default']['username'] = "root";
-$db['default']['password'] = "password";
+$db['default']['username'] = "ahmad";
```

Comment:
Please don't change the default password

Reply:
sorry, now i'm changed this file to default password